src/buildCommon.js

```diff
@@ -774,7 +774,22 @@ const staticSvg = function(value: string, options: Options): SvgSpan {
     return span;
 };
 
+const binrelClass = (arg) => {
+    // \binrel@ spacing varies with (bin|rel|ord) of the atom in the argument.
+    // (by rendering separately and with {}s before and after, and measuring
+    // the change in spacing).  We'll do roughly the same by detecting the
+    // atom type directly.
+    const atomType = (arg.type === "ordgroup" &&
+        arg.value.length ? arg.value[0].type : arg.type);
+    if (/^(bin|rel)$/.test(atomType)) {
+        return "m" + atomType;
+    } else {
+        return "mord";
+    }
+};
+
```

Comment:
I think this should be in `src/functions/mclass.js` since it's mclass-related or in `src/defineFunction.js` like `ordargument`.

Reply:
True, 2 out of 3 uses are in `src/functions/mclass.js`.  I originally wrote it this way, but switched for some reason.  Switched back.  This PR should be ready for review now.